packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -138,6 +158,14 @@ class ExpiringMultiPartyClient {
       this.emp.methods.getCurrentTime().call()
     ]);
 
+    if (this.contractType === "Perpetual") {
+      this.latestCumulativeFundingRateMultiplier = this.toBN(
+        (await this.emp.methods.fundingRate().call()).cumulativeMultiplier.rawValue
+      );
+    } else {
+      this.latestCumulativeFundingRateMultiplier = this.fixedPointAdjustment;
```

Comment:
If `latestCumulativeFundingRateMultiplier` defaults to `fixedPointAdjustment` perhaps this class var is better described as `numTokenMultiplier`. Otherwise setting `latestCumulativeFundingRateMultiplier = fixedPointAdjustment` is a bit weird.

Reply:
well `fixedPointAdjustment` simply is `1e18`. So this means that it sets the `cumlativeFundingRateMultipler` to 1 if the contract type is EMP.

`numTokenMultiplier` is a good name but I feel that it's better to keep this really explicit. What it _really_ is the cumulative funding rate multiplier and so it makes it really explicit by keeping the var named accordingly. 

WDYT?

Reply:
I'm fine leaving as it is to reduce churn